app/controllers/application_controller/queries.rb

```diff
@@ -0,0 +1,437 @@
+# frozen_string_literal: true
+
+# see application_controller.rb
+# rubocop:disable Metrics/ModuleLength
+module ApplicationController::Queries
```

Comment:
```suggestion
module ApplicationController::Queries # rubocop:disable Metrics/ModuleLength
```
So you don't have to disable it at the end. 2 fewer physical lines in the file.

Reply:
I like the inline cop disabling, but Rubocop doesn't seem to allow inline disabling class definitions.

(Also - note that because these are "magic" comments, they don't count towards the class line length. Comments and blank lines don't count.)